data/water/gsl-water-level-2016/index.html

```diff
@@ -0,0 +1,82 @@
+---
+author:
+  display_name: UGRC Staff
+  email: ugrc@utah.gov
+tags:
+  - history
+  - lakes
+  - water
+  - gsl
+categories: []
+date: 2022-05-03 12:00:00 -0600
+GSL2016:
+  hub:
+    name: Utah Great Salt Lake Water Level 2016
+    item_id: 015723bdb92447f788471450c8fcab16
+  updates:
+    - 2016
+Bonneville:
+  hub:
+    name: Utah Historic Lake Bonneville
+    item_id: fb95503ddf904428af3f308eca0eff6e
+  updates:
+    - 2007
+---
+<figure class="caption caption--right">
+ <a href="{% link images/GSL2016_lg.png %}">
+  <img class="caption__image" title="map" src="{% link images/GSL2016.png %}"
+    alt="UtahGreatSaltLakeWaterLevel2016" loading="lazy" /></a>
+  <figcaption class="caption__text">GSL Water Level 2016</figcaption>
+</figure>
+{% include abstract.html
+name="Great Salt Lake Resources"
+stewards="UGRC"
+abstract="Various resources related to the Great Salt Lake." %}
+
+<div>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://wasatchweatherweenies.blogspot.com/2022/04/the-great-salt-finger-lake.html">The Great Salt Finger Lake (2022)</a></li>
```

Comment:
I think we could do without this one since it's not a data link or state/local government resource. It feels unusual for us to link to a single post on a personal blog.

Reply:
i agree